clients/python/girder_client/__init__.py

```diff
@@ -29,6 +29,17 @@
 import shutil
 import six
 import tempfile
+from six.moves.configparser import SafeConfigParser
+
+_config_defaults = {
+    'host': 'localhost',
+    'scheme':  'http',
+    'apiRoot': '/api/v1'
+}
+CONFIG_DIR = os.environ.get(
+    'XDG_CONFIG_HOME', os.path.join(os.path.expanduser('~'), '.config'))
```

Comment:
+1

We could also put `girder-cli.conf` in a top level `girder` directory under `CONFIG_DIR`, though I can't think of a use case for this at the moment.